source/module.c

```diff
@@ -368,6 +368,8 @@ napi_status aws_napi_dispatch_threadsafe_function(
     });
     /* Must always decrement the ref count, or the function will be pinned */
     napi_status release_status = napi_release_threadsafe_function(tsfn, napi_tsfn_release);
+    /* main thread can exit now */
+    AWS_NAPI_ENSURE(env, napi_unref_threadsafe_function(env, tsfn));
```

Comment:
This is not needed. It came from some testing I had from staging repo that I tired to fix the crash about threadsafe function. The public repo already have a better fix for the crash. Seems like I forgot to remove it from staging repo.